main/src/cgeo/geocaching/search/SuggestionsAdapter.java

```diff
@@ -48,7 +50,8 @@ public View newView(final Context context, final Cursor cursor, final ViewGroup
     }
 
     public void changeQuery(@NonNull final String searchTerm) {
-        changeCursor(query(searchTerm));
+        //get new cursor asynchronous since this might take a while
+        AndroidRxUtils.andThenOnUi(AndroidRxUtils.networkScheduler, () -> query(searchTerm), this::changeCursor);
```

Comment:
Although I have no idea, what the purpose of these different Schedulers are: Is `AndroidRxUtils.networkScheduler` really the most fitting one?

Reply:
Maybe we could even create a new `DbScheduler` if we want to move all DB calls into the background...

Reply:
@fm-sys we might do this. But DB calls should already be done in the background in all places (as it is best practice to put all IO operations away from the main thread because they might be long-running). This is maybe not completely implemented in c:geo but should be... 

So this would not be just about creating a scheduler but to establish a concept for putting DB calls in the background everywhere across c:geo. This is definitely worth its own isssue.

Also I don't even understand the meaning of the existing schedulers and when to use which. So the first step to approach this would be to clarify their meaning and when they should be used.